mail_tracking/README.rst

```diff
@@ -93,6 +93,27 @@ If you want to see all tracking emails and events you can go to
 * Settings > Technical > Email > Tracking emails
 * Settings > Technical > Email > Tracking events
 
+When the message generates and 'error' status, it will apear on discuss 'Failed'
+channel. Any view that uses 'mail_thread' widget can show the failed messages
+too.
+
```

Comment:
I miss here an explanation about the filter added in all documents for showing those with any failed message (or most explicitly, with any failed message not reviewed)